public/src/components/channelManagement/bannerTests/bannerVariantPreview.tsx

```diff
@@ -198,17 +199,63 @@ const useStyles = makeStyles(({ palette }: Theme) => ({
     fontStyle: 'italic',
     fontSize: '20px',
   },
+  controlsContainer: {
+    position: 'fixed',
+    backgroundColor: palette.grey[100],
+    borderRadius: '4px',
+    top: spacing(3),
+    left: spacing(3),
+    padding: spacing(3),
+  },
 }));
 
 interface BannerVariantPreviewProps {
   variant: BannerVariant;
   design?: BannerDesign;
+  shouldShowTickerToggle?: boolean;
+}
+
+const DEFAULT_TICKER_SETTINGS: TickerSettingsWithData = {
```

Comment:
I wonder if instead of hardcoding this in here we could include `tickerSettings` in the variant data prop? And let `useTickerData` do its thing

Reply:
I've refactored to support this - most of the complexity around toggling the ticker is now outside of the `BannerVariantPreview` component. When toggled on for a design the ticker data is threaded through as part of the variant (as usual).